database/migrations/2019_01_17_134449_create_tnms_alarms_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateTnmsAlarmsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('tnmsalarms', function (Blueprint $table) {
+            $table->increments('id');
+            $table->unsignedInteger('alarm_num');
+            $table->unsignedInteger('device_id')->index('device_id');
+            $table->integer('neID')->index('neID');
```

Comment:
neID is unneeded remove it
Add a field called:
`tnmsne_info_id` to relate to the tnmsneinfo entry (the ne owning the alarm)